examples/nspawn2.yaml

```diff
@@ -0,0 +1,7 @@
+---
+graph: mygraph
+resources:
+  nspawn:
+  - name: mgmt-nspawn1
```

Comment:
I suppose changing it to 2 would be better. That way during the demo, we can live edit the file to change the state, which is pretty cool since it will notice live.